src/land/ascat_ssm2ioda.py

```diff
@@ -88,6 +92,7 @@ def _read(self):
         vals = vals.astype('float32')
         lats = lats.astype('float32')
         lons = lons.astype('float32')
+        vals = vals*0.01
```

Comment:
See the question above.

Reply:
Scaled from zero to one, added an inline comment here.